src/components/audio-player-view/audio-player-view.tsx

```diff
@@ -123,10 +123,13 @@ const AudioPlayerView = Event.withEventManager(
           const pluginConfig: AudioPlayerConfig = player.plugins['audioPlayer'].config;
           const showMorePluginsIcon: boolean = availablePlugins.length > 0;
 
-          useEffect(() => {
-            addPlayerClass!();
+          useLayoutEffect(() => {
             eventManager.listen(player, core.EventType.CHANGE_SOURCE_ENDED, _handleMediaMetadata);
             eventManager.listen(player, core.EventType.PLAYER_RESET, _handleMediaMetadataReset);
+          }, []);
+
+          useEffect(() => {
```

Comment:
can addPlayerClass be called the useLayoutEffect or does it need to be called separately inside useEffect ?
does it need to happen before or after this component is rendered ?